app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    function, file_in_name, file_out_name = command.split()
+    if (function == "cp"
+            and file_in_name.endswith(".txt")
+            and file_out_name.endswith(".txt")
+            and file_in_name != file_out_name):
+
+        with (open(file_in_name, "r") as file_in,
```

Comment:
How do you handle file-related errors that may occur here?
Also you can use `shutil.copy` here.